prepare_oim/roles/prepare_oim_validation/tasks/main.yml

```diff
@@ -25,5 +30,36 @@
 - name: Validate high availability config
   ansible.builtin.include_tasks: validate_high_availability.yml
 
+- name: Include composable role data
+  ansible.builtin.include_tasks: include_composable_roles_config.yml
+  when: enable_oim_ha | default(false)
+
+- name: Include mapping details if OIM HA mapping is used
+  when: 
+    - enable_oim_ha | default(false)
+    - oim_ha_group | length > 1
+    - oim_passive_node_mapping
+  block:
```

Comment:
instead of doing all this here, should we configure passive oim in discovery role only for pcs?

@priti-parate